### PR TITLE
150 adding tool version and absolute paths in pheval run parameters

### LINKED ISSUE #150: Makefile: absolute paths, and tool version in pheval run command
Makefile must be generated with a `pheval run` command with `--version` passing the tool version and using absolute paths in `--input-dir` and `--output-dir` parameters. Otherwise, 'pheval run' doesn't work properly.

Stacktrace representing the absence of absolute paths in pheval run parameters

---

**command**

```bash
pheval run \
 --input-dir ./configurations/exomiser-13.2.0-default \
 --testdata-dir /home/vinicius/workspace/pheval/corpora/exomiser/corpus2/scrambled0.3 \
 --runner exomiserphevalrunner \
 --tmp-dir data/tmp/ \
 --version 13.2.0 \
 --output-dir ./results/exomiser-13.2.0-default/corpus2-scrambled0.3
```
---
**stacktrace**:
```python
Traceback (most recent call last):
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/bin/pheval", line 6, in <module>
    sys.exit(pheval())
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/vinicius/workspace/pheval/src/pheval/cli_pheval.py", line 90, in run
    runner_instance.run()
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/pheval_exomiser/runner.py", line 48, in run
    run_exomiser(
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/pheval_exomiser/run/run.py", line 208, in run_exomiser
    run_exomiser_local(
  File "/home/vinicius/.cache/pypoetry/virtualenvs/pheval-B0LyEAGE-py3.10/lib/python3.10/site-packages/pheval_exomiser/run/run.py", line 118, in run_exomiser_local
    for file in all_files(tool_input_commands_dir)
  File "/home/vinicius/workspace/pheval/src/pheval/utils/file_utils.py", line 25, in all_files
    files = [path for path in directory.iterdir()]
  File "/home/vinicius/workspace/pheval/src/pheval/utils/file_utils.py", line 25, in <listcomp>
    files = [path for path in directory.iterdir()]
  File "/usr/lib/python3.10/pathlib.py", line 1017, in iterdir
    for name in self._accessor.listdir(self):
FileNotFoundError: [Errno 2] No such file or directory: 'results/exomiser-13.2.0-default/corpus2-scrambled0.3/tool_input_commands
```

### PR DESCRIPTION
**Pull request by souzadevinicius**
fixes #150

### CODE REVIEW

File: resources/Makefile.j2 [hunk 1]
```diff
@@ -63,14 +64,14 @@ corpora/{{ run.tool }}/{{ run.corpus }}/scrambled{{ run.corpusvariant }}/corpus.
...
-	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single \
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/lirical \
 	 --output-dir $(shell pwd)/$(shell dirname $@)/vcf
 {% endif %}
 	test -d $(shell dirname $@)/phenopackets || mkdir -p $(shell dirname $@)/phenopackets
 	pheval-utils scramble-phenopackets \
 	 --scramble-factor {{ run.corpusvariant }} \
 	 --output-dir $(shell pwd)/$(shell dirname $@)/phenopackets \
-	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/single
+	 --phenopacket-dir=$(shell pwd)/$(TEST_DATA)/phenopackets/lirical
```
Comment (matentzn) [inline resources/Makefile.j2:74]: The `phenopacket-dir` should be tied to the dependencies of this make goal..

File: resources/Makefile.j2 [hunk 2]
```diff
@@ -49,11 +49,12 @@ semsim-ingest: configurations/{{ config.tool }}-{{ config.version }}/{{ config.c
 results/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/{{ run.corpus }}-scrambled{{ run.corpusvariant }}/results.yml: configurations/{{ run.tool }}-{{ run.version }}-{{ run.configuration }}/config.yaml
 	mkdir -p $(shell dirname pwd)/$(shell dirname $@)
 	pheval run \
-	 --input-dir {{ directories.config }}/{{ run.tool }}-{{ run.version }}-{{ run.configuration }} \
+	 --input-dir $(shell pwd)/{{ directories.config }}/{{ run.tool }}-{{ run.version }}-{{ run.configuration }} \
```
[the remark this answers is not in the document]
Comment (souzadevinicius) [reply resources/Makefile.j2:52]: Yep, it's a good solution ;)

**Review by matentzn — CHANGES_REQUESTED** [review]
Some minor changes! 

AMAZING work, super complex issue, and well done.

File: corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json
Comment (yaseminbridges) [inline corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json:1]: Is it possible to revert the phenopackets back to the original state? When the `files` section is empty

File: resources/pheval-config.yaml [hunk 3]
```diff
@@ -14,7 +14,33 @@ directories:
...
-#when we run the make preparation (not default) we should create symlink if they don't exists in default 
+#when we run the make preparation (not default) we should create symlink if they don't exists in default
+
+corpora:
+  - id: lirical
+    scrambled:
+      - factor: 0.5
+      - factor: 0.7
+    custom_variants:
+      - id: no_phenotype
```
Comment (yaseminbridges) [inline resources/pheval-config.yaml:25]: What does the `custom_variants` mean?

File: corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json
Comment (yaseminbridges) [reply corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json:1]: All of the LIRICAL phenopackets now have an added `files` section, e.g.,

```yaml
 "files": [
     {
       "uri": "file:///home/vinicius/workspace/pheval/corpora/lirical/scrambled-0.5/vcf/Abdul_Wahab-2016-GCDH-Patient_5.vcf.gz",
       "fileAttributes": {
         "fileFormat": "vcf",
         "genomeAssembly": "GRCh37"
       }
     }
   ],
```

I had removed this section before in an earlier PR. It looks to be only the LIRICAL phenopackets that have this section added and moved- the rest have stayed the same and just changed location.

File: resources/pheval-config.yaml (hunk 3 above)
Comment (yaseminbridges) [reply resources/pheval-config.yaml:25]: Ah, got it!

File: corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json
Comment (souzadevinicius) [reply corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json:1]: Is there a step that overwrites the URI attribute?
Yes, I can revert them, but ideally, pheval shouldn't change these values, right?

Comment (yaseminbridges) [reply corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json:1]: There isn't a step that overwrites the URI attribute a new `File` object will be created. I think we want PhEval to add the reference of the VCF as it's related to the sample, plus,  I use the genome assembly information and VCF file name from here.

Comment (matentzn) [reply corpora/lirical/default/phenopackets/Abdul_Wahab-2016-GCDH-Patient_5.json:1]: @souzadevinicius do you undestand what needs to be done here? Not following the convo. Is this a matter of updating the PR from `main` branch?

**Review by matentzn — APPROVED** [review]

**Pull request merged**